Library/Homebrew/cmd/config.rb

```diff
@@ -148,6 +148,9 @@ def describe_system_ruby
   end
 
   def describe_java
+    # java_home doesn't exist on all OS Xs; it might be missing on older versions.
+    return "N/A" unless File.executable? "/usr/libexec/java_home"
```

Comment:
Is it maybe worth doing `which java` (or `which java_home`) here instead in case it's manually installed? Would be useful on `Linuxbrew` too.